### DISCUSSION TITLE
Welcome to pyglotaran Discussions!

### DISCUSSION THREAD

**Discussion by jsnel** [glotaran/pyglotaran#732] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas, suggestions for improvements or links to your work!
  * Examples of how you use pyglotaran or other related tools from the Glotaran ecosystem are welcome!
  * Engage with other community members.
  * Welcome others and are open-minded.

**Comment by farbowitz** [comment]
Hi Joris,

I was curious if pyglotaran might be able to handle more involved models that can't be summarized in a k-matrix. Specifically, what I'm thinking of is exciton-exciton annihilation, which depends on the square of the population rather than being linear. I'm currently working on this concept, but would love any thoughts you have on the idea.

Best,
Dan

**Reply by jsnel** [reply]
Hi Daniel @farbowitz, 

Thank you for your question! I'm sorry I didn't notice your question before (somehow GitHub did not send me a notification of this reply).

You have asked a very interesting question, and I think it's best if dr. Ivo van Stokkum (@ism200 ) first answers your question in general terms, since he has the most experience with this topic and can provide you with a reference or two. I notified him of your question and he indicated he might be able to write an answer next week (busy schedule permitting). 

Then the @glotaran/developers can perhaps follow up that with some pyglotaran specific details on how one might realize this. 

Kind regards,

Joris @jsnel
